x/staking/src/types/tx.rs

```diff
@@ -15,6 +15,9 @@ use prost::Message;
 use serde::{Deserialize, Serialize};
 use std::str::FromStr;
 
+// constant used in flags to indicate that description field should not be updated
+pub const DO_NOT_MODIFY_DESCRIPTION: &str = "[do-not-modify]";
```

Comment:
You could simply remove field from cli

Reply:
Why? What happens if the user wants to change the field?

Reply:
But why we need this weird message then? Nothing actually prevents from usage of these options. Why not make them optional without any weird description that this value shouldn't be modified.

Reply:
Agreed to use domain type to isolate [do-not-modify] part.